Project/feature-u-soundboard/src/features/common/pagination/component/Pagination.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react/react-in-jsx-scope */
+
+import { Pagination } from '@mui/material';
+import React from 'react';
+
+export interface Props {
+  data: Array<any>;
+}
+
+export default function PaginationFeature(props: Props) {
+  const handleChange = (event, value: number) => {
+    window.location.assign(`?page=${value}`);
+  };
+
+  const urlSearchParams = new URLSearchParams(window.location.search);
```

Comment:
Ik snap niet waarom je hier de search bij haalt. Ik zou zeggen dat je dat hier niet wilt doen. Ik zou ervanuit gaan dat data al gefiltered is door search.